types/jquery/JQuery.d.ts

```diff
@@ -1780,7 +1780,9 @@ $( "div" ).children( ".selected" ).css( "color", "blue" );
 </html>
 ```
      */
-    children(selector?: JQuery.Selector): this;
+    children<K extends keyof HTMLElementTagNameMap>(selector: K): JQuery<HTMLElementTagNameMap[K]>;
+    children<K extends keyof SVGElementTagNameMap>(selector: K): JQuery<SVGElementTagNameMap[K]>;
+    children(selector?: JQuery.Selector): JQuery;
```

Comment:
```suggestion
    children(selector?: JQuery.Selector): this;
```
to keep with convention in file?

Reply:
oh, sorry, I misread the intent/fixed topic.

Reply:
no problem. Is it fine?